portfolio/src/main/webapp/index.html

```diff
@@ -14,6 +14,7 @@ <h1>Jacob's Portfolio</h1>
       <a class="active" href="#home">HOME</a>
       <a href="faq.html">FAQ</a>
       <a href="contact.html">CONTACT</a>
+      //<a href="/data">DataServlet</a> 
```

Comment:
Does this actually comment in HTML? Also, generally we shouldn't include comments like this in production code. And if we do then it deserves a little context.